client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts

```diff
@@ -43,11 +44,17 @@ const useInteractionsTab = (input: useInteractionsTabInput) :  useInteractionsTa
         // TODO: delete the event from db
     }
 
+    const loadInteractions = () => {
+        // TODO: add loading from DB
+        axios.get('/intersections?epidemioligyNumber=111').then(res => console.log(res));
```

Comment:
console .log 🤷‍♀️

Reply:
It is a stupid axios request to get information from the be

Reply:
it is a stub that will be replaced